test/conf/options.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @fileoverview the default options for different dirs
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+// TODO: add your
+export const defaultOptions = [
+  { dir: 'test/__snapshot__/parser/', parserOptions: {}, printerOptions: {} },
```

Comment:
you can config your default options here.

Reply:
can you add 'allowTypes' to this test cases and regenerate all the snapshots?

https://github.com/kataw/kataw/tree/main/test/__snapshot__/parser/declarations/type/arrow-function-type

Reply:
Did you fix this?

Reply:
https://github.com/kataw/kataw/issues/102

Reply:
Give me the exact test. I can't reproduce this